docs/tutorials/local-gpu.md

```diff
@@ -1,47 +1,21 @@
 # Setting up a Local GPU Environment
 
-Marin was mainly developed on Google Cloud Platform (GCP) using TPUs.
-However, it is possible to run Marin on GPUs as well.
-
 This guide will walk you through the steps to set up a local GPU environment for Marin.
-By "local", we mean a machine that you run directly, as opposed to using Ray's autoscaler to launch a cluster of GPU nodes.
+By "local", we mean a machine that you run jobs on directly, as opposed to using Ray's autoscaler to launch a cluster of GPU nodes.
 Similar steps will let you run Marin on a cloud GPU environment using Ray's autoscaler, but we defer that to a future guide.
 
 ## Prerequisites
 
-This part walks you through having the following installed:
+Make sure you've followed the [installation guide](installation.md) to do the basic installation.
+
+In addition to the prerequisites from the basic installation, we have GPU-specific dependencies:
 
-- Python 3.10 or higher
-- pip (Python package manager)
-- Git
 - CUDA Toolkit (version 12.1 or higher)
 - cuDNN (version 9.1 or higher)
 
-## Basic system setup
```

Comment:
If we're going to give basic instructions, we should do it properly in installation (these don't work for me).